charts/rhdh/templates/rolling-demo-sidecars-job.yaml

```
@@ -26,76 +26,25 @@ spec:
               TEMP_DIR=$(mktemp -d)
               PATCH_FILE="${TEMP_DIR}/patch.json"
 
-              # Create the container definition files
-              cat > "${TEMP_DIR}/location.json" <<EOF
+              cat > "${TEMP_DIR}/road-core-sidecar.json" <<EOF
               {
-                "name": "location",
+                "name": "road-core-sidecar",
                 "env": [
-                  { "name": "NORMALIZER_FORMAT", "value": "JsonArrayFormat" },
-                  { "name": "POD_IP", "valueFrom": { "fieldRef": { "fieldPath": "status.podIP" } } },
-                  { "name": "POD_NAMESPACE", "valueFrom": { "fieldRef": { "fieldPath": "metadata.namespace" } } }
+                  { "name": "PROJECT", "value": "rhdh" },
+                  { "name": "RCS_CONFIG_FILE", "value": "/app-root/config/rcsconfig.yaml" },
+                  { "name": "RHDH_CONFIG_FILE", "value": "/app-root/config/app-config-rhdh.yaml" }
                 ],
                 "envFrom": [
-                  { "secretRef": { "name": "rhdh-rhoai-bridge-token" } },
-                  { "secretRef": { "name": "ai-rh-developer-hub-env" } }
+                  { "secretRef": { "name": "team-cluster-key" } },
```

```suggestion
                  { "secretRef": { "name": "lightspeed-secrets" } },
```

you need to add this to use the OLLAMA_TOKEN and OLLAMA_URL

---

Shouldn't it have `envFrom` to use the secrets in the sidecar container?